test_loader.cfc

```diff
@@ -368,17 +368,22 @@ component extends="mxunit.framework.TestCase" {
 				}
 			}
 		];
-		for ( var name in tests ) {
-			var test = tests[name];
-			// call method under test
-			var result = variables.loader.load(cfc = test.args.cfc, data = test.args.data);
-			// check assertions
-			AssertEquals(test.expect.returnType, GetMetaData(result).name, "#name# - return type doesn't match expected");
-			AssertEquals(
-				test.expect.serialized,
-				DeserializeJson(SerializeJson(result)),
-				"#name# - result did not contain expected data"
-			);
+		for ( var dataType in ["json", "struct"] ) {
+			for ( var name in tests ) {
+				var test = tests[name];
+				if ( dataType == "json" ) {
+					test.args.data = SerializeJson(test.args.data);
+				}

```

Comment:
This is the update that tests the new change. I just wrapped the original tests in a loop over `["json", "struct"]`. So, it runs the tests like it did before with structs _and_ it runs the tests with JSON.